website/content/en/docs/best-practices/resource-pruning.md

```diff
@@ -0,0 +1,149 @@
+---
+title: "Resource pruning"
+linkTitle: "Resource Pruning"
+weight: 3
+description: Recommendations for pruning resources
+---
+
+## Overview
+
+Operators can create [Jobs][jobs] or Pods as part of their normal operation and when those Jobs or Pods
+complete, they can remain on the Kubernetes cluster if not specifically removed.  These resources
+can consume valuable cluster resources like disk storage (e.g. etcd).  The resources here we 
+are discussing are ones that are not tied to a Custom Resource using an ownerReference.
+
+Operator authors have traditionally had two pruning options:
+
+ * leave the resource cleanup to a system admin to perform
+ * implement some form of pruning within their operator solution
+
+For our purposes when we say, *prune*, we mean to remove a resource (e.g. kubectl delete) from
+a kubernetes cluster for a given namespace.
+
+This documentation describes the pattern and library useful for implementing a solution within an operator.
+
+## operator-lib prune library
+
+A simple pruning implementation can be found in the [operator-lib prune package][operator-lib-prune].  This 
+package is written in golang and is meant to be used within golang based operators.  This package was 
+developed to include common pruning strategies as found in common operators.  The package also allow 
+for customization of hooks and strategies.
```

Comment:
Using language similar to the other docs. We always reference Go operators as Go-based operators https://sdk.operatorframework.io/docs/building-operators/golang/quickstart/

```suggestion
A simple pruning implementation can be found in the [operator-lib prune package][operator-lib-prune]. This 
package is written in Go and is meant to be used within Go-based operators. This package was 
developed to include common pruning strategies as found in common operators. The package also allow 
for customization of hooks and strategies.
```